docs/ref/models/instances.txt

```diff
@@ -553,6 +554,11 @@ keep the parent model's data.
 
     The ``keep_parents`` parameter was added.
 
+.. versionchanged:: 1.9
+
+    Returns the number of objects deleted
```

Comment:
more simply, "The return value describing the number of objects deleted was added." (it's already described in more detail above)